ansible/roles_ocp_workloads/ocp4_workload_nginx_ingress/.yamllint

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
There is an `ansible/roles_ocp_workloads/.yamllint` so no additional yamllint files are needed in subdirectories.